server/src/main/resources/reference.conf

```diff
@@ -30,7 +30,7 @@ broccoli {
   # How to connect to Nomad
   nomad {
     # The URL of the Nomad client to be used by Broccoli.
-    url = "http://localhost:4646"
```

Comment:
Please do not change the default value from `localhost`. If you need custom configuration, please provide a specific `application.conf`. For more infos on how configuration is read, you can check https://github.com/lightbend/config#standard-behavior

Reply:
10x, will look later